src/config/development.js

```diff
@@ -2,7 +2,7 @@
 import { TX_SERVICE_HOST, SIGNATURES_VIA_METAMASK, RELAY_API_URL } from '~/config/names'
 
 const devConfig = {
-  [TX_SERVICE_HOST]: 'https://safe-transaction.staging.gnosisdev.com/api/v1/',
+  [TX_SERVICE_HOST]: 'https://safe-transaction-service.dev.gnosisdev.com/api/v1/',
```

Comment:
Why dev? I thought dev only exists for development of tx service

Reply:
It was `dev` because it requires the refactored version of the backend. Can be changed back now since it is in staging